.travis.yml

```diff
@@ -4,9 +4,14 @@ go: "1.10"
 services:
 - docker
 
+before_install:
+- go get golang.org/x/tools/cmd/goimports
+# install operator-sdk
+- sudo curl -JL https://github.com/operator-framework/operator-sdk/releases/download/v0.8.0/operator-sdk-v0.8.0-x86_64-linux-gnu -o /usr/bin/operator-sdk
+- sudo chmod +x /usr/bin/operator-sdk
 install: true
 script:
-- make test 
+- make check
```

Comment:
Can you add after this line.

```bash
if [[ -n "$(git status --porcelain)" ]] ; then echo "It seems like you need to run make. Please run it and commit the changes"; git status --porcelain; false; fi
```

Reply:
No need to, `make check` is read-only and it raises an error if something is not up-to-date.